src/pages/ExamsPage/components/AttemptList.jsx

```diff
@@ -1,9 +1,138 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, DataTable } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
 
-const AttemptList = () => (
-  <div data-testid="attempt_list">
-    Placeholder for attempt list
-  </div>
+const capitalizeFirstLetter = (string) => string.charAt(0).toUpperCase() + string.slice(1);
+
+const ResetAction = (props) => (
+  <Button onClick={() => console.log('Reset', props)}>
+    Reset
+  </Button>
 );
 
+const AttemptList = ({ attempts }) => {
+  const { formatMessage, formatDate } = useIntl();
+
+  return (
+    <div>
+      <p>Data Table</p>
+      <DataTable
+        isPaginated
+        isSelectable
+        initialState={{
+          pageSize: 20,
+        }}
+        isSortable
+        itemCount={attempts.length}
+        bulkActions={[
+          <ResetAction />,
+        ]}
+        additionalColumns={[
+          {
+            id: 'action',
+            Header: formatMessage({
+              id: 'AttemptsList.action',
+              defaultMessage: 'Action',
+              description: 'Action to reset the exam attempt',
+            }),
+            // TODO: For MST-1945, make this button actually "reset" (i.e. delete)
+            // the exam attempt for the cell's respective row after confirmation.
+
+            // NOTE: This line throws lots of errors. Docs for this are from:
+            // https://paragon-openedx.netlify.app/components/datatable/#custom-cell-content
+            Cell: ({ row }) => <Button variant="link" size="sm" onClick={() => console.log(`Resetting ${row.values.name}`)}>Reset</Button>,
+          },
+        ]}
+        data={attempts}
+        columns={[
+          {
+            Header: formatMessage({
+              id: 'AttemptsList.exam_name',
+              defaultMessage: 'Exam Name',
+              description: 'The name of the exam attempted',
```

Comment:
I think a more accurate description of this would be 'Header for table column listing exam name' or the like. Same goes for all of these since out of context a translator won't know this is the header and not the actual name of the field.